### PR TITLE
ISSUE-33 add scroll to overlays

### LINKED ISSUE #33: Allow scrolling in the overlays
We currently hide text on overlays which don't meet minimum width & height values. This won't always work as text size can change by browser / site / accessibility settings, leading to overflow:
image
Instead of a minimum height, we should enable scrolling within the overlay.

### PR DESCRIPTION
**Pull request by tabanliviu**
fixes #33

### CODE REVIEW

**Comment by Ian-Hi** [comment]
Verified ok

File: app/public.js [hunk 1]
```diff
@@ -137,6 +137,7 @@
     element.style.left = `${image.position.left}px`;
     element.style.backgroundColor = getBackgroundColor(getImageCoverage(image));
     element.style.border = '1px solid black';
+    element.style.overflow = 'auto';
```
Comment (MalcolmSmithUK) [inline app/public.js]: @tabanliviu MAINTAINABILITY - Please move static to CSS if possible.

**Review by MalcolmSmithUK — CHANGES_REQUESTED** [review]

**Comment by tabanliviu** [comment]
@MalcolmSmithUK ready for review

**Comment by tabanliviu** [comment]
@MalcolmSmithUK ready for review

**Review by MalcolmSmithUK — APPROVED** [review]

**Pull request merged**